corehq/apps/reports/models.py

```diff
@@ -384,7 +385,8 @@ def url(self):
             return url_base + '?' + self.query_string
         except UnsupportedSavedReportError:
             return "#"
-        except Exception:
+        except Exception as e:
+            logging.error(e.message)
```

Comment:
what are you hoping to accomplish with this? any reason not to use `logging.exception` which will also give you a stack trace and what not?